cellprofiler/gui/pipelinelistview.py

```diff
@@ -142,7 +142,7 @@ def __init__(self, panel, frame):
         self.outputs_panel = wx.Panel(panel)
         outputs_sizer.Add(self.outputs_panel, 1, wx.EXPAND)
         self.outputs_panel.SetSizer(wx.BoxSizer())
-        self.outputs_panel.SetBackgroundStyle(wx.BG_STYLE_COLOUR)
+        self.outputs_panel.SetBackgroundStyle(wx.BG_STYLE_PAINT)
```

Comment:
@bethac07 @0x00b1 @AetherUnbound it seems like BG_STYLE_COLOUR is no longer supported even though it's still listed in the documentation (https://wxpython.org/Phoenix/docs/html/wx.BackgroundStyle.enumeration.html); could you please let me know which background style is preferred?